cfme/tests/automate/test_simulation.py

```diff
@@ -86,3 +88,64 @@ def test_assert_failed_substitution(copy_class):
             request="Call_Instance",
             execute_methods=True,
         )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[1445089])
+def test_automate_simulation_result_has_hash_data(custom_instance):
+    """
+    The UI should display the result objects if the Simulation Result has
+    hash data.
+
+    Bugzilla:
+        1445089
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Create a Instance under /System/Request called ListUser, update it so that it points
+               to a ListUser Method
+            2. Create ListUser Method under /System/Request, paste the Attached Method
+            3. Run Simulation
+        expectedResults:
+            1.
+            2.
+            3. The UI should display the result objects
+    """
+    from . import user_list_hash_data
+    instance = custom_instance(ruby_code=user_list_hash_data)
+
+    # Executing automate method
+    with LogValidator(
+            "/var/www/miq/vmdb/log/automation.log",
+            matched_patterns=['.*User List.*:id=>1, :name=>"Fred".*']).waiting(timeout=120):
+
+        simulate(
+            appliance=instance.appliance,
+            attributes_values={
+                "namespace": instance.klass.namespace.name,
+                "class": instance.klass.name,
+                "instance": instance.name,
+            },
+            message="create",
+            request="Call_Instance",
+            execute_methods=True,
+        )
+    view = instance.create_view(AutomateSimulationView)
+    assert (
+        view.result_tree.click_path(
+            *(
```

Comment:
you wrap a bunch of values into tuple and then roll it out into bunch of parameters again. what for ?

Reply:
After executing this test case. This type of tree generates in simulation tree view. I just have to check if this tree exists or not. Hence I am providing this path to check if it click able or not.
Please let me know if you have some other way to perform this part. 
Actually, I could have tried ```ha_path``` but I preferred clicking in that path.

Reply:
click_path is correct. I'm talking about *() constuction. you wrap items into tuple and then roll out this tuple back to items.

Reply:
Updated as below:
```
assert (
        view.result_tree.click_path(
                f"ManageIQ/SYSTEM / PROCESS / {instance.klass.name}",
                f"ManageIQ/System / {instance.klass.name} / Call_Instance",
                f"{instance.domain.name}/System / {instance.klass.name} / {instance.name}",
                "values",
                "Hash",
                "Key",
        ).text
        == "Key"
    )```